rllib/utils/typing.py

```diff
@@ -11,11 +11,17 @@
     Union,
 )
 
+import sys
 import numpy as np
 import gym
 
 from ray.rllib.utils.annotations import ExperimentalAPI
 
+if sys.version_info >= (3, 10):
+    from typing import TypeAlias
+else:
+    from typing_extensions import TypeAlias
```

Comment:
rather than hardcoding a version string here, should we simply:
```
try:
    from typing import TypeAlias
except ImportError:
    from typing_extensions import TypeAlias?
```

Reply:
Sure, changed it! I think I used the Python version because I had seen that pattern elsewhere in the source code but this seems more robust.